examples/extended_validation/src/tests.rs

```diff
@@ -0,0 +1,86 @@
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::{ContentType, Status};
+
+use super::rocket;
+
+fn test_login<T>(user: &str, pass: &str, age: &str, status: Status, body: T)
+    where T: Into<Option<&'static str>>
+{
+    let rocket = rocket();
+    let query = &format!("username={}&password={}&age={}", user, pass, age);
+
+    let mut req = MockRequest::new(Post, "/login")
+        .header(ContentType::Form)
+        .body(&query);
+
+    let mut response = req.dispatch_with(&rocket);
+    assert_eq!(response.status(), status);
+    if let Some(string) = body.into() {
+        assert!(response.body().and_then(|body| body.into_string()).map_or(false, |s| s.contains(string)));
+    }
+}
+
+const OK: Status = self::Status::Ok;
```

Comment:
Just do a `use rocket::http::Status::{Ok, BadRequest, UnprocessableEntity}` at the top and remove this `const`.